### PR TITLE
feat: add documentation for revoking did documents

### PR DESCRIPTION
**Pull request by cavacado**
context: https://www.pivotaltracker.com/story/show/176973225
this pr does the following:
- extends the tutorial for `DID` documents for the revocation steps
- misc changes that do not reflect the current state of revocation of `DID` document

deployed here: https://deploy-preview-64--laughing-cori-52b566.netlify.app/

### CODE REVIEW

File: website/docs/verifiable-document/comparison.md [hunk 1]
```diff
@@ -39,9 +39,11 @@ For DID, it's the opposite, everything is private.
 
 Revocation is part of our Ethereum Smart Contracts.
 
-If you use DID, there is for the moment no way to revoke a document. We have ideas on [how to provide such a feature](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly), but the work is still in progress. In any case,
+We have ideas on [how to provide such a feature](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly), but the work is still in progress. In any case,
```
Comment (Nebulis) [inline website/docs/verifiable-document/comparison.md:42]: You can get rid of this part of it's not relevant anymore

However it must be added that it you use revocation then did is not free anymore, need to pay for the deployment of revocation store

File: website/docs/verifiable-document/comparison.md [hunk 2]
```diff
@@ -39,9 +39,11 @@ For DID, it's the opposite, everything is private.
...
 DID will have to rely on any kind of [CRL](https://en.wikipedia.org/wiki/Certificate_revocation_list) or even on a document store (which bring back the need to pay with ethers).
 
+As of *180221*, its possible to revoke a document if a document store has been declared in its revocation block. refer [here](/docs/verifiable-document/did/revoking-document) for the steps.
```
Comment (Nebulis) [inline website/docs/verifiable-document/comparison.md:45]: Let's just update the doc to reflect the current state of oa

File: website/docs/verifiable-document/did/revoking-document.md [hunk 3]
```diff
@@ -3,4 +3,52 @@ title: Revoking Documents
 sidebar_label: Revoking Documents
 ---
 
-As of today, it's not possible to revoke a signed document but we are working on it. You can follow [our progress](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly).
+As of today *180221*, its possible to revoke a signed document only if you fulfill certain **prerequisites**. Any future progress can be followed [here](https://github.com/Open-Attestation/adr/blob/master/issuing_using_did.md#for-documents-that-are-signed-directly).
+
+## Prerequisites:
+- an `ethr` DID (if you've been following the above steps, you should have one already)
```
Comment (Nebulis) [inline website/docs/verifiable-document/did/revoking-document.md:9]: We also need a document store deployed, as this part doesn't explain how to do it

I would suggest to link to deploy a document store

Comment (cavacado) [reply website/docs/verifiable-document/did/revoking-document.md:9]: ah yea okay

File: website/docs/verifiable-document/comparison.md (hunk 2 above)
Comment (cavacado) [reply website/docs/verifiable-document/comparison.md:45]: 👌

**Review by Nebulis — APPROVED** [review]

**Pull request merged**